src/V3EmitCSyms.cpp

```diff
@@ -156,6 +156,15 @@ class EmitCSyms final : EmitCBaseVisitor {
         string out = scpname;
         // Remove hierarchy
         string::size_type pos = out.rfind('.');
+
+        // If the last identifier is escaped, move to that esc in case there's a '.' in it
+        std::string::size_type esc = scpname.rfind('\\');
```

Comment:
Shouldn't this look for space backslash?  There can be a backslash that is itself escaped.

Reply:
So the lrm says ```Escaped identifiers shall start with the backslash character (\) and end with white space (space, tab, newline). They provide a means of including any of the printable ASCII characters in an identifier (the decimal values 33 through 126, or 21 through 7E in hexadecimal).``` 

So the space goes after the escaped ident. It is true that this does catch // in the ident, incorrectly. I could search for '.\', but then if the prev is escaped as well it needs to be ' \' . However both of these would catch '\ \' in an ident and '\.\'. In order for a complete solution I would need to loop through the whole string. So I think the best option is to check for '\' in the string and then do the slow path.